sklearn/cluster/tests/test_mean_shift.py

```diff
@@ -103,9 +107,11 @@ def test_parallel():
     assert_array_equal(ms1.labels_, ms2.labels_)
 
 
-def test_meanshift_predict():
+def test_meanshift_predict(global_dtype):
     # Test MeanShift.predict
+    global X
     ms = MeanShift(bandwidth=1.2)
+    X = X.astype(global_dtype)
```

Comment:
That looks bad :D
you are modifying it globally so the next tests will use the modified version
```suggestion
    ms = MeanShift(bandwidth=1.2)
    X = X.astype(global_dtype)
```

Reply:
Indeed, that's insanely bad. :smile: